WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsViewModel.swift

```diff
@@ -2,21 +2,58 @@ import UIKit
 import Yosemite
 
 /// The Product Settings contains 2 sections: Publish Settings and More Options
-struct ProductSettingsViewModel {
+final class ProductSettingsViewModel {
 
-    let sections: [ProductSettingsSectionMediator]
+    public private(set) var sections: [ProductSettingsSectionMediator] {
```

Comment:
I don't think you need the `public`. 🙂 I thought `public` is generally only used by libraries. 

```suggestion
    private(set) var sections: [ProductSettingsSectionMediator] {
```